mex/common/transform.py

```diff
@@ -78,9 +78,21 @@ def kebab_to_camel(string: str) -> str:
 
 
 @lru_cache(maxsize=1024)
-def normalize(string: str) -> str:
-    """Normalize the given string to lowercase, numerals and single spaces."""
-    return " ".join(re.sub(r"[^a-z0-9]", " ", string.lower()).split())
+def camel_to_split(string: str) -> str:
+    """Convert the given string from `CamelCase` into `Split Case`."""
+    return " ".join(
+        word
+        for word in re.split(r"([A-Z]+(?![a-z])|[a-z]+|[A-Z][a-z]+)", string)
+        if word.strip()
+    )
+
+
+@lru_cache(maxsize=1024)
+def split_to_camel(string: str) -> str:
+    """Convert the given string from `Split Case` into `CamelCase`."""
+    if len(tokens := re.split(r"\s+", string)) > 1:
+        return "".join(word.title() for word in tokens if word)
+    return string[:1].upper() + string[1:] if string else string
```

Comment:
```suggestion
    return string.title()
```
why wanting to keep possible upper case letters in the middle of the word here?
is the `if string else string` needed, if the function only accepts str (no None) anyway?